docs/user/monitoring/kibana-alerts.asciidoc

```diff
@@ -81,8 +81,8 @@ by running checks on a schedule time of 1 minute with a re-notify interval of 6
 [[kibana-alerts-large-shard-size]]
 == Large shard size
 
-This alert is triggered if a large (primary) shard size is found on any of the 
-specified index patterns. The trigger condition is met if an index's shard size is 
+This alert is triggered if a large average shard size (across associated primaries) is found on any of the 
+specified index patterns. The trigger condition is met if an index's (primary average) shard size is 
```

Comment:
Let's just say "if an index's average shard size" -- we mentioned we are only using primary in the parenthesis and I think that's enough, beyond that I think it's more confusing to see "primary" show up more.